public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataTransformations/PanelDataTransformationsTab.tsx

```diff
@@ -0,0 +1,161 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import { DragDropContext, Droppable } from 'react-beautiful-dnd';
+
+import { DataTransformerConfig, GrafanaTheme2, IconName } from '@grafana/data';
+import { selectors } from '@grafana/e2e-selectors';
+import { SceneObjectBase, SceneComponentProps, SceneDataTransformer } from '@grafana/scenes';
+import { Button, ButtonGroup, ConfirmModal, Container, CustomScrollbar, useStyles2 } from '@grafana/ui';
+import { TransformationOperationRows } from 'app/features/dashboard/components/TransformationsEditor/TransformationOperationRows';
+
+import { VizPanelManager } from '../../VizPanelManager';
+import { PanelDataPaneTabState, PanelDataPaneTab } from '../types';
+
+import { EmptyTransformationsMessage } from './EmptyTransformationsMessage';
+
+interface PanelDataTransformationsTabState extends PanelDataPaneTabState {}
+
+export class PanelDataTransformationsTab
+  extends SceneObjectBase<PanelDataTransformationsTabState>
+  implements PanelDataPaneTab
+{
+  static Component = PanelDataTransformationsTabRendered;
+  tabId = 'transformations';
+  icon: IconName = 'process';
+  private _panelManager: VizPanelManager;
+
+  getTabLabel() {
+    return 'Transformations';
+  }
+
+  getItemsCount() {
+    const dataProvider = this._panelManager.state.panel.state.$data;
+    if (dataProvider instanceof SceneDataTransformer) {
+      return dataProvider.state.transformations.length;
+    } else {
+      return null;
+    }
```

Comment:
Think we can do sth similar to what we did for queries tab

```ts
getItemsCount() {
  return this.getQueries().length;
}
```

The getQueries method uses queryRunner getter under the hood that will throw if there's no query runner for a panel. That saves us checks for the object existance or whether or not it's of a correct type.